### ISSUE TITLE
remove edit and replace with list selection [BLOCKED by file list needed]

### ISSUE THREAD

**Issue by SimLeek** [SimLeek/cse360_text_file_analyzer#6]
Java has click and double click function overrides. If we want to keep a list of files on the left, we can override the doubleclick methods, get the filename, open the file, and get the text.

I'm not sure how the edit button would work now... it seems like it might've been automatically taken care of by some java gui stuff, kind of like browse.

**Comment by SimLeek** [comment]
Done. creating a new issue though.
